inc/azure_c_shared_utility/platform.h

```diff
@@ -17,6 +17,9 @@ extern "C" {
     MOCKABLE_FUNCTION(, const IO_INTERFACE_DESCRIPTION*, platform_get_default_tlsio);
     MOCKABLE_FUNCTION(, STRING_HANDLE, platform_get_platform_info);
 
+    void platform_set_http_proxy(const char* proxyHostnamePort, const char* proxyUsernamePassword);
```

Comment:
If declared in platform.h, should these 2 functions defined for all platforms. For platforms that do not use tlsio_openssl.c, these functions are not defined.

Reply:
fixed, thanks!